src/Nethermind/Nethermind.Core/Transaction.cs

```diff
@@ -197,7 +197,9 @@ private void ClearPreHashInternal()
         /// </summary>
         public int GetLength(ITransactionSizeCalculator sizeCalculator, bool shouldCountBlobs)
```

Comment:
Might be cool in addition to Transaction to have NetworkTransaction class-wrapper with GetTransaction method or so to make it clearer which form is used here or there. So you can't call GetLength for transaction without NetworkWrapper. May be too much work though